packages/adblocker-benchmarks/string.js

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright (c) 2017-present Cliqz GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+const _N = (strings, number) => strings[0] + number.toLocaleString() + strings[1];
+
+module.exports = {
+  _N,
+
+  _T: (strings, time) => {
+    if (time === Infinity || time === -Infinity) {
+      time = NaN;
+    }
+
+    const formatted = isNaN(time) ? 'n/a' : _N`${Math.round(time * 1e6) / 1e3} μs`;
```

Comment:
The goal here is to show the time in microseconds but also with up to three decimal places.